charts/operator-wandb/charts/flat-runs-field-updater/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v2
+name: flat-runs-field-updater
```

Comment:
Could we name this `flat-run-fields-updater` (same with the name of this file) to match what we have in SaaS?